brainstormforce/astra
# AST-2999 - fix: z-index lowered for tooltip due to UI conflict

### Description

- Fix: z-index lowered for tooltip due to UI conflict
- https://brainstormforce.atlassian.net/browse/AST-2999

### Screenshots

https://d.pr/i/91L0EF

### Types of changes


- Fix: z-index lowered for tooltip due to UI conflict



### How has this been tested?

- Open Customizer > Woocommerce > Single Product
- Check the tooltip for any setting, should not overlap section title.

### Checklist:
